tests/test_corner_names.py

```
@@ -0,0 +1,230 @@
+        s += 1.0
+    return s
+
+
+def parse_corners_ini(content: str | None) -> dict[int, dict]:
+    by_id: dict[int, dict] = {}
+    if not content:
+        return by_id
+    current_id: int | None = None
+    for raw in content.splitlines():
+        line = raw.strip()
+        if not line or line.startswith(";") or line.startswith("#"):
```

Ruff (0.15.9)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)